### ISSUE TITLE
[Shifted Base BatchNorm] BatchNorm Layer 코드 분석 후 발표

### ISSUE THREAD

**Issue by ssaru** [DeepBaksuVision/Binarized_Neural_Network#9]

**Comment by ssaru** [comment]
2019.11.23일까지 아래 자세히 보고 올 것

1. Batch Normalization Implementation
https://wiseodd.github.io/techblog/2016/07/04/batchnorm/
